### ISSUE TITLE
[Accessibility] PeoplePicker: Unable to focus search button with keyboard

### ISSUE THREAD

**Issue by steventmayer** [OfficeDev/office-ui-fabric-react#1522]
With the PeoplePicker (should be all pickers that have this search button), I am unable to focus the Search button/functionality with just the keyboard. To get around this, I am thinking of creating a separate search button that will push the search button suggestions internally. Am I missing something, or any other suggestions?

**Comment by mdahamiwal** [comment]
>should be all pickers that have this search button

@steventmayer, I can't find a search button in PeoplePicker component. https://dev.office.com/fabric#/components/PeoplePicker. It would help if you can provide an example or screenshot of it. Thanks

**Comment by steventmayer** [comment]
@mdahamiwal The search button can be found in the [Suggestions](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/pickers/Suggestions/Suggestions.tsx) component of the Pickers.  If you pass searchForMore text, it'll display a search button.  Here is a [snapshot](http://imgur.com/FC3u3Zv) - it looks like the code is plumbed for it, it's just not using the focusSearchForMoreButton function that is available.
